src/main/java/com/limechain/sync/warpsync/runtime/Runtime.java

```diff
@@ -0,0 +1,36 @@
+package com.limechain.sync.warpsync.runtime;
```

Comment:
I think the Runtime logic shouldn't be inside `warpsync` folder. It's a separate module itself.